apps/charterafrica/src/lib/ecosystem/ecosystem/index.js

```diff
@@ -5,6 +5,9 @@ import {
   processContributors,
   processTools,
   processOrganisations,
+  updateContributor,
```

Comment:
If these new methods are updateXXXX, should we call the old ones createXXXX? Or what does processXXXX stand for?

Reply:
processXXX processes data from Airtable together with github data using airtableId and github ID then stores this data in payload.
updateXXX on the other hand fetches data for every item from github and updates the collection accordingly.

Reply:
I know what they do @koechkevin ... I'm talking about naming. To add, I'm ok with updateXXXX. Isn't:
>processXXX processes data from Airtable together with github data using airtableId and github ID then stores this data in payload.
>
Just mean createXXXX or prepareXXXX?